ui/src/assets/json/form-builder-list-sample.json

```diff
@@ -0,0 +1,91 @@
+{
+  "sample": [
+    {
+      "type": "text",
+      "label": "Headings",
+      "options": { "tag": "h1", "defaultValue": "This is a heading" },
+      "key": "483e7d80c",
+      "model": "text_483e7d80c",
+      "order": 5
+    },
+    {
+      "type": "text",
+      "label": "Paragraph",
+      "options": { "tag": "p", "defaultValue": "This is a text" },
+      "key": "c7fbb6f2a",
+      "model": "text_c7fbb6f2a",
+      "order": 6
+    },
+    {
+      "type": "slider",
+      "label": "Slider",
+      "question": "slider question",
+      "description": "slider description",
+      "options": { "defaultValue": 0, "min": 0, "max": 100, "step": 1 },
+      "key": "109f3d239",
+      "model": "slider_109f3d239",
+      "order": 4
+    },
+    {
+      "type": "checkbox",
+      "label": "Multiple choices",
+      "question": "checkbox question",
+      "description": "checkbox description",
+      "options": {
+        "defaultValue": ["option_1", "option_2"],
+        "options": [
+          { "value": "option_1", "text": "option 1" },
+          { "value": "option_2", "text": "option 2" },
+          { "value": "option_3", "text": "option 3" }
+        ]
+      },
+      "key": "e977226e6",
+      "model": "checkbox_e977226e6",
+      "order": 3
+    },
+    {
+      "type": "radio",
+      "label": "Single choice",
+      "question": "radio question",
+      "description": "radio description",
+      "options": {
+        "defaultValue": "Option_2",
+        "options": [
+          { "value": "Option_2", "text": "Option 2" },
+          { "value": "Option_3", "text": "Option 3" }
+        ]
+      },
+      "key": "c617faead",
+      "model": "radio_c617faead",
+      "order": 2
+    },
+    {
+      "type": "input",
+      "label": "Long open answer",
+      "question": "textarea question",
+      "description": "textarea description",
+      "options": {
+        "defaultValue": "textarea",
+        "placeholder": "",
+        "type": "textarea"
+      },
+      "key": "bb583bcd7",
+      "model": "input_bb583bcd7",
+      "order": 1
+    },
+    {
+      "type": "input",
+      "label": "Short open answer",
+      "question": "input text question",
+      "description": "",
+      "options": {
+        "defaultValue": "",
+        "placeholder": "input text placeholder",
+        "type": "text"
+      },
+      "key": "4be14b2cb",
+      "model": "input_4be14b2cb",
+      "order": 0
+    }
```

Comment:
The current db design does not distinguish between "short open answer" and "long open answer", so the back-end and db guys should add one column in the open answer class/table like a boolean "isLongOpenAnswer"; for the "options" field i am not sure but maybe they can do some tricks like putting the defaultValue in the answer field right away. Please communicate these 2 needs (distinguish short and long open answers, options) with the back-end and db guys. Please, communicate this with the rest of the team.

Reply:
Lets wait for the others until I merge this MR; just in case I need to delete the textarea XD

Reply:
Based on the recent team discussion, please remove the "long open answer" from ui and json structure. We will only use "short open answer" and call it simply "open answer". Thanks a lot!

Reply:
Alright, I will work on it and notify you later when I finished.

Reply:
![image](https://user-images.githubusercontent.com/12537724/173197807-91309a48-153a-4a0e-b919-97ff33b23135.png)
 Updated.